addons/mail/static/src/models/activity/activity.js

```diff
@@ -36,7 +38,7 @@ registerModel({
                 data2.id = data.id;
             }
             if ('note' in data) {
-                data2.note = data.note;
+                data2.note = data.note && markup(data.note);
```

Comment:
Doing it here it is not a good idea, if it is written from a different flux it will not have the `markup`. Should do something similar to `prettyBody`/`contentAsMarkup` instead.